assets/js/blocks/stock-filter/utils.ts

```diff
@@ -23,8 +23,9 @@ export const getActiveFilters = (
 		? params.split( ',' )
 		: ( params as string[] );
 
-	return Object.keys( filters ).filter( ( filter ) =>
-		parsedParams.includes( filter )
+	const filter_keys = Object.keys( filters );
+	return parsedParams.filter( ( param ) =>
+		filter_keys.includes( param )
 	);
```

Comment:
To ensure we are following the correct naming conventions, I'd recommend updating the variable name to `filterKeys` instead and also updating the format to clear out all JS lint errors.
```suggestion
	const filterKeys = Object.keys( filters );

	return parsedParams.filter( ( param ) => filterKeys.includes( param ) );
	);
```